coalib/coala.py

```diff
@@ -35,9 +33,12 @@ def main():
         args = default_arg_parser().parse_args()
 
         if args.show_bears:
+            from coalib.settings.ConfigurationGathering import (
+                get_filtered_bears)
             local_bears, global_bears = get_filtered_bears(
                 args.filter_by_language, log_printer)
 
+            from coalib.output.ConsoleInteraction import show_bears
             show_bears(local_bears,
                        global_bears,
                        args.show_description or args.show_details,
```

Comment:
A question about the code below, what is `show_capabilities` ? I can't seem to find it in DefaultArgParser

Don't you need to import `get_filtered_bears` there (In the elif) also ?